test/Microsoft.AspNetCore.SignalR.Tests/EndToEndTests.cs

```diff
@@ -107,11 +107,49 @@ public EndToEndTests(ServerFixture<Startup> serverFixture, ITestOutputHelper out
 
                     Assert.Equal(bytes, buffer.Array.AsSpan().Slice(0, result.Count).ToArray());
 
+                    logger.LogInformation("Closing socket");
+                    await ws.CloseOutputAsync(WebSocketCloseStatus.Empty, "", CancellationToken.None).OrTimeout();
                     logger.LogInformation("Waiting for close");
-                    result = await ws.ReceiveAsync(buffer, CancellationToken.None).OrTimeout();
+                    result = await ws.ReceiveAsync(buffer, CancellationToken.None).OrTimeout(TimeSpan.FromHours(1));
```

Comment:
Uhhhhh.... Is it worth a timeout at all at that point?

Reply:
This was testing 😄 I shall remove it.